mne/preprocessing/artifact_detection.py

```diff
@@ -4,12 +4,87 @@
 
 
 import numpy as np
-
 from ..annotations import (Annotations, _annotations_starts_stops)
 from ..transforms import (quat_to_rot, _average_quats, _angle_between_quats,
                           apply_trans, _quat_to_affine)
+from ..filter import filter_data
 from .. import Transform
-from ..utils import (_mask_to_onsets_offsets, logger)
+from ..utils import (_mask_to_onsets_offsets, logger, verbose)
+
+
+@verbose
+def annotate_muscle(raw, threshold=1.5, picks=None, min_length_good=.1,
+                    filter_freq=[110, 140], verbose=None):
+    """Detect segments with muscle artifacts.
+
+    Detects segments periods that contains high frequency activity beyond the
+    specified threshold. Muscle artifacts are most notable in the range of
+    110-140 Hz.
```

Comment:
do you have a ref for this?  My experience doing ICA on EEG told that it can be in the 30Hz -> 40Hz range.

Reply:
Different muscles have different characteristic frequencies. I believe that 30-40Hz is related to more frontal/facial muscles, while 110-140Hz is related more to larger posterior muscles controlling head posture.
This has a reasonable summary paragraph : https://www.frontiersin.org/articles/10.3389/fnhum.2013.00138/full#h7

Reply:
I have similar experience to @agramfort - muslce artifacts are usually broadband enough that can be seen far below 110 Hz. I couldn't find any mention of a frequency range in the ref you gave, maybe its in a different fragment of the text?

Reply:
Sorry didn't mean to link to a particular section.
Here is a better link https://www.frontiersin.org/articles/10.3389/fnhum.2013.00138/full#h2 its the second paragraph titled "The Characteristics of Muscle Artifacts in MEG/EEG"

Reply:
Thanks, but that part is in line with what @agramfort said - all muscle groups mentioned ther together span a wide range between 15 and 100 Hz.

Reply:
what do you propose to say? 
"Muscle artifacts are **more** **detectable** in the range of 110-140 Hz, although they span a wide range between 15 and 100 Hz"? In MEG, we compared the PSD of sections with and without muscle artifacts a the PSD diverged at 45 Hz and onwards. 
The reason for the 110-140 Hz is that it works well for detecting muscle activity.

Reply:
I'd say that although muscle artifacts span [here some wide range] we use 110-140 in this tutorial because it seems to work well [in general / for this specific dataset] (and we could link the paper you gave).
BTW fieldtrip tutorial uses this specific frequency range: 110 - 140. They only say:

> For instance, muscle EMG is mostly expressed in the higher frequencies of the EEG/MEG signal. A bandpassfilter between e.g. 110 and 140 would make use of this knowledge and will result in channels showing a large power amplitude when your subject made a muscle twitch.

In general I think it would be better to use a dataset with more artifacts (see the main discussion thread) - on this dataset we could check if 110-140 works ok. If it does we'd just say that 110-140 seems to work well for detection of muscle artifacts. But due to various sampling frequency of different datasets its good to highligh that muscle artifacts could be detected using other/wider frequency ranges so if someone has a lower sfreq they can still use this function with good results.

Reply:
Oh, sorry, I lost track - it is not a text for the tutorial. Then I'd change the freq range to also work for lower sfreq files. Maybe `filter_freq=(60, None)` or `(80, None)` where None is set to nyquist? I'd also just say that muscle artifacts cover a wide frequency range but are more distinguishable in higher frequencies (without reference to specific frequency range).